File: c-sharp/at-csharp-wellington-falcao/Exercicio08/Gerente.cs
namespace Exercicio08
{
    public class Gerente : Funcionario
        
    {

        // Construtor padrão
        public Gerente() { }

        // Construtor da classe Gerente que herda de Funcionario e aplica um aumento de 20% no salário base
        public Gerente(string nome, string cargo, double salarioBase)
            : base(nome, cargo, salarioBase * 1.2)
        {        
        }

        // Método para exibir os dados do gerente, sobrescrevendo o método da superclasse Funcionario
        public override void ExibirSalario()
        {
            base.ExibirSalario();
        }
    }
}

File: java/at-java-wellington-falcao/src/br/edu/infnet/at/ex08/Gerente.java
package br.edu.infnet.at.ex08;

public class Gerente extends Funcionario{
	
	
	public Gerente (String nome, double salarioBase) {
		super(nome, salarioBase);
	}
	
	@Override
	public double calcularSalario() {
		return salarioBase * 1.20;
	}
	
	@Override
	public void mostrarSalario() {
		System.out.println("O salário do Gerente %s é R$ %.2f.".formatted(nome, calcularSalario()));
	}
	
}
